Update Readme.md (#8)
Rwpulley/Portfolio: CarpetCleaning/Readme.md

# Carpet Cleaning MS-SQL Database

## Summary

> This is a Database for a Carpet cleaning company. It can be used in multiple other fields as well.\

Reviewer: ```suggestion
This is a Database for a Carpet cleaning company. It could be used for other industries as well.\
```
